client/src/components/app/app.js

```diff
@@ -1,26 +1,34 @@
-import React from 'react';
-import './app.css';
-
+import React, { useState } from 'react';
 import { BrowserRouter as Router } from 'react-router-dom';
 
+import { UserContext, ServiceContext } from '../context';
 import { navigation } from '../router/navigation';
+import TestService from '../../service/TestService';
+
 import Header from '../header/Header';
+import ErrorBoundry from '../error-boundary/ErrorBoundary';
 
-const isAuth = false;
-const user = {
-  isExpert: false,
-  isAdmin: false,
-};
+import './app.css';
 
 const App = () => {
-  const { routes, links } = navigation(isAuth, user);
+  const [user, setUser] = useState(null);
```

Comment:
![image](https://user-images.githubusercontent.com/22274137/112715978-b7198700-8ef4-11eb-8c8e-acfc4a3f7114.png)
we can move this logic into this context, make sense?

Reply:
resolved